### DISCUSSION TITLE
How to use the EDC for Connector as a Service?

### DISCUSSION THREAD

**Discussion by MikhailGordienk** [eclipse-edc/.github#42] [Q&A]
Hello!

As far as I know, there were plans to support such a concept as "Connector as a Service" (let it be CaaS) for those clients who can't / don't want to run their own instance of the Connector.

I have the following questions:

* How storage permissions will be handled? I see two options:

	- CaaS will have its own managed storage and each consumer will be granted a quota in it. Consumers will download data via CaaS.
	- Consumers will register their storage (S3 buckets) granting CaaS full access (so it could write the requested data)

* Will CaaS look like a single-consumer connector for others? I am thinking about tenant management when someone asks CaaS about its artifacts, consumers might want to use CaaS only to download artifacts and not to share ones.

**Comment by jimmarino** [comment]
CaaS would not be something that we (as an open-source project) would build. A vendor could take the EDC a built a CaaS solution on top of the open-source platform. There are probably many different ways of solving this, but those would be outside the open-source project.
